pkg/issuer/vault/issue.go

```diff
@@ -314,3 +317,11 @@ func (v *Vault) vaultTokenRef(name, key string) (string, error) {
 
 	return token, nil
 }
+
+func ipAddressesToString(ipAddresses []net.IP) []string {
```

Comment:
This PR already defines `IPAddressesToString` in `pkg/util/util.go` - I don't think we need this second definition of the same function here?